include/sel4runtime.h

```diff
@@ -50,17 +50,17 @@ char const *const *sel4runtime_envp(void);
 auxv_t const *sel4runtime_auxv(void);
 
 /*
- * Get the address of the TLS base register.
+ * Get the address of the TLS region
```

Comment:
These changes shouldn't be made. The 'TLS base register' refers to the platform register used to indicate the location of the TLS (even if that register doesn't refer to the 'base'). I think this change makes the description misleading as to what these function do.

Reply:
The TLS base register doesn't have an address.  It's a register.  None of the supported architectures allow the address of a register to be taken.   Which is why I changed the comment : if it's wrong, what does the function *really* do?

Reply:
Maybe say 'stored in the TLS base regsiter'